scope

```diff
@@ -70,6 +71,20 @@ check_not_running() {
     esac
 }
 
+# Run `weave expose` if it's not already exposed.
+weave_expose() {
+    weave=$(which weave)
```

Comment:
The more I think about this, the more I think its a good idea, because (1) consistency and (2) dns is moving inside the router, with the weave script being update for that - so if we just depend on the weave script, we don't need to change anything.